slemicro/README.md

```diff
@@ -366,6 +370,112 @@ Options:
  -o		(Optional) Path to the tweaked-SLE-Micro-SelfInstall.iso file (./tweaked.iso by default)
  -d		(Optional) Disk device where SLE Micro will be installed (if not provided, the first one that the installer finds)
 ```
+## create_vm_with_eib.sh
+
+This script creates a SLE Micro VM on OSX/Linux using UTM (or Libvirt) based on EIB.
+
+The script will output the virtual terminal to connect to (using `screen` if needed) as well as the
+IP that it gets from the OSX DHCPD service.
+
+### Prerequisites
+
+* `podman`
+* `yq`
+* `qemu-img`
+
+NOTE: They can be installed using `brew`.
+* `qemu-img` is available via the `qemu` package.
+
+* [UTM 4.2.2](https://docs.getutm.app/) or higest (required for the scripting part)
+
+* EIB configuration file and folder already created, including the SL Micro raw image.
+  * Download the raw image file from the SUSE website at https://www.suse.com/download/sle-micro/
+  * Select ARM or X86_64 architecture (depending on the Operating system of the host)
+  * Look for the raw file (I.e.- `SL-Micro.aarch64-6.0-Default-GM2.raw`)
+  * Note: SLE Micro RT image can be used as well.
+
+### Enviroment variables
+
+It requires a few enviroment variables to be set to customize it, the bare minimum is just:
+
+```
+# Folder where the VM will be hosted
+VMFOLDER="${HOME}/VMs"
+```
+
+The rest of them can be observed in the script itself.
+
+They can be stored in the script basedir as `.env` or in any file and use the `-f path/to/the/variables` flag.
+
+**NOTE**: There is a `vm*` pattern already in the `.gitignore` file so you can conviniently name your VM parameters file as `vm-foobar` and they won't be added to git. 
+
+NOTE:
+1. EIB vars and settings are not verified, EIB will complain by itself if needed.
+2. The EIB config file is currently hardcoded as `eib.yaml`
+3. The EIB folders and files need to be precreated by the user
+
+### Usage
+
+For a simple example like:
+
+```bash
+$ tree --noreport eib 
+eib
+└── smolvm
+    ├── base-images
+    │   └── SL-Micro.aarch64-6.0-Default-GM2.raw
+    └── eib.yaml
+
+$ cat eib/smolvm/eib.yaml
+apiVersion: 1.1
+image:
+  imageType: raw
+  arch: aarch64
+  baseImage: SL-Micro.aarch64-6.0-Default-GM2.raw
+  outputImageName: eib-image.raw
+operatingSystem:
+  rawConfiguration:
+    diskSize: 30G
+  users:
+    - username: root
+      encryptedPassword: $6$G392FCbxVgnLaFw1$Ujt00mdpJ3tDHxEg1snBU3GjujQf6f8kvopu7jiCBIhRbRvMmKUqwcmXAKggaSSKeUUOEtCP3ZUoZQY7zTXnC1
```

Comment:
Perhaps mention the password so people can easily copy/paste this and/or how to generate the hash?

Reply:
Honestly I just copied it from the official docs https://documentation.suse.com/suse-edge/3.1/html/edge/quickstart-eib.html and I don't even know which password is :D Lemme fix it.